src/lib/collab/provider.spec.js

```diff
@@ -2,10 +2,6 @@ import { CollabProvider } from './provider'
 import { getVersion, sendableSteps } from 'prosemirror-collab'
 import isEqual from 'lodash/isEqual'
 
-jest.mock('prosemirror-collab', () => {
-  return { getVersion: jest.fn(), sendableSteps: jest.fn() }
```

Comment:
i am not sure it's a good idea to move it inside jest.setup. If it's useful for one test only, let's leave it here

Reply:
@trollepierre actually it was used in two files that's why I moved it

Reply:
ok,then, where was the other deletion?

Reply:
https://github.com/cozy/cozy-notes/pull/320/files#diff-6bc52ecd51d546c11ea78b3b0ef47f66f850c9368a4bcdeeb2356bc1e9772b28L4

https://github.com/cozy/cozy-notes/pull/320/files#diff-7f9b120b4cffec96953a4b8d23c0a15b674c96d16fd5188a3b1fcb1c1caeac11L5